### PR TITLE
97804 application overview - model changes, new models, service changes

### PR DESCRIPTION
**Pull request by MarkRobinsonDfe**
Dfe.Academies.External.Web Changes:-
1) New models for application overview
2) Amended ConversionApplication model
3) Amended AcademisationWorkflowEnums - extra enum
4) Additional methods on ConversionApplicationRetrievalService

Dfe.Academies.External.Web.UnitTest Changes:-
1) New tests for new models
2) Amended ConversionApplication model tests
3) Amended ConversionApplicationRetrievalService tests

### CODE REVIEW

File: Dfe.Academies.External.Web/Models/SchoolOrSchoolsApplyingToConvert.cs [hunk 1]
```diff
@@ -2,7 +2,7 @@
 {
     public class SchoolOrSchoolsApplyingToConvert
     {
-        public int? Id { get; set; }
-        public string? SchoolOrSchoolsApplyingToConvertProperty { get; set; }
+        public long? Id { get; set; }
+        public string? SchoolName { get; set; }
```
Comment (wystanpalm-madetech) [inline Dfe.Academies.External.Web/Models/SchoolOrSchoolsApplyingToConvert.cs:6]: 👍

File: Dfe.Academies.External.Web/Models/ConversionApplicationContributor.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    public ConversionApplicationContributor(string name, string role)
+    {
+        Name = name;
+        Role = role;
+    }
+
+    public long Id { get; set; }
+    public string Name { get; set; }
+
+    public string Role { get; set; }
```
Comment (wystanpalm-madetech) [inline Dfe.Academies.External.Web/Models/ConversionApplicationContributor.cs:14]: Do we have an enum for Role? I know there's only Chair in the roles enum so far but I think we might need to challenge David on that.

**Review by wystanpalm-madetech — APPROVED** [review]
Some thoughts here, I don't think they need to block merging though so I'm approving.

File: Dfe.Academies.External.Web/Models/ConversionApplicationContributor.cs (hunk 2 above)
Comment (MarkRobinsonDfe) [reply Dfe.Academies.External.Web/Models/ConversionApplicationContributor.cs:14]: yeah we do, i didn't realize it was the same until I was looking at another page this afternoon

**Review by wystanpalm-madetech — APPROVED** [review]

**Pull request merged**